src/genericSchemeRegistry/index.ts

```diff
@@ -96,40 +96,57 @@ export class ActionField {
     this.placeholder = options.placeholder;
     this.transformation = options.transformation;
   }
-  /**
-   * the value to pass to the contract call (as calculated from the user's input data)
-   * @return [description]
-   */
-  public callValue(userValue: string|string[]) {
-    if (Array.isArray(userValue)) {
-      userValue = userValue.map((val: string) => Object.prototype.hasOwnProperty.call(val, "trim") ? val.trim() : val);
-    } else if (Object.prototype.hasOwnProperty.call(userValue, "trim")) {
-      userValue = userValue.trim();
+
+  private transformValue(value: string, web3: any): string | boolean {
+
+    if (Object.prototype.hasOwnProperty.call(value, "trim")) {
+      value = (value as string).trim();
     }
 
     if (this.type === "bool") {
-      return parseInt(userValue as string, 10) === 1;
+      return parseInt(value as string, 10) === 1;
     }
-
-    if (this.decimals) {
-      return (new BN(userValue as string).mul(new BN(10).pow(new BN(this.decimals)))).toString();
+    /**
+     * Note that if this is an array item, the field's one `decimals` value applies
+     * to all items in the array.
+     */
+    else if (this.decimals) {
+      return (new BN(value as string).mul(new BN(10).pow(new BN(this.decimals)))).toString();
     }
+    else {
+      switch (this.transformation) {
+        case "namehash": {
+          return namehash.hash(value);
```

Comment:
why these additional spaces ?

Reply:
I'm not sure what spaces you are referring to here.  I know there were some added elsewhere in the file, I don't know whether that was my editor's preference or lint.  But they seem reasonable for readability.

Reply:
It might be your editor .can you try to remove it please ? see https://github.com/daostack/alchemy/pull/2289/files#diff-3f9aa40293ee5ff0e8a044c6f87937328712efc794fabbf134cb807019b09b5bR33. also

Reply:
ok I removed the whitespace changes